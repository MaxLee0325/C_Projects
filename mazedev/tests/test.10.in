Alpha
END
END
Alpha
Tango 
END
