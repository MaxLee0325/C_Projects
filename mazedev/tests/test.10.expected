Entering room Alpha.
Oops: Cannot find room Tango.
